tests/acceptance/features/apiOcm/share.feature

```diff
@@ -24,8 +24,8 @@ Feature: an user shares resources using ScienceMesh application
       | shareType       | user       |
       | permissionsRole | Viewer     |
     Then the HTTP status code should be "200"
-    When using server "REMOTE"
-    And user "Brian" lists the shares shared with him without retry using the Graph API
+    And using server "REMOTE"
```

Comment:
Why is there `And` after `Then` when these two steps aren't similar?  `using server "REMOTE" doesn't look like a Then step for me